### ISSUE TITLE
GSoC 2016 Refactor storyboard

### ISSUE THREAD

**Issue by bphenriques** [systers/malaria-app-ios#31]
Storyboard as the app grows will hard to manage (specially constraints and themes). During the process I will also suggest investigating https://github.com/robb/Cartography

**Comment by manas15** [comment]
In the PR which I made I have solved this issue by
Creating dividing the current storyboard file into 4 storyboard files using storyboard reference technique.. Following are the new storyboards 
1. Main: which has our rootViewController, setupViewController and other storyboard references 
2. Pages: which has everything to do with 1st tab (all the pages we have in the project )
3.Trip:  This storyboard has PlanTripViewController and ListItemViewController
4.Info:  This has the info hub viewController and PostDetails View Controller 
This will solve 2 major problems:
1. This will help more developers to work on different things together without having merge conflicts. By Working on different storyboards.
2. This will keep things more organized as the project grows.
